docker/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     environment:
       DB_HOST: database
     healthcheck:
-      test: curl -s -o /dev/null -w "%{http_code}" http://cdash:8080/ping | grep 200 > /dev/null
+      test: curl -s -o /dev/null -w "%{http_code}" http://cdash:80/ping | grep 200 > /dev/null
```

Comment:
This healthcheck doesn't work for a production docker compose installation now.

Experimenting locally, I had to do:
`curl -k https://cdash:80/ping`

...but the `-k` may be specific to this particular instance.